SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -246,7 +248,7 @@ Worker_RequestId USpatialSender::CreateEntity(USpatialActorChannel* Channel)
 
 	if (SpatialSettings->bEnableUnrealLoadBalancer)
 	{
-		ComponentDatas.Add(AuthorityIntent::CreateAuthorityIntentData(NetDriver->LoadBalanceStrategy->GetLocalVirtualWorkerId()));
+		ComponentDatas.Add(AuthorityIntent::CreateAuthorityIntentData(VirtualWorkerTranslator->GetLocalVirtualWorkerId()));
```

Comment:
It seems that we've introduced the dependency on `VirtualWorkerTranslator` only to pass a `uint32` value here. I was wondering if we are going to have more reasons to have `VirtualWorkerTranslator` in SpatialSender in the future?

Reply:
Not that I am aware. If there is a nicer way to do this then lets do that.

Reply:
I looked into ways we might be able to remove it, but I think we really do need it. We can revisit later if the NetDriver refactor lets us make a better interface to the loadbalancing components.

Reply:
Ok, resolving.